### ISSUE TITLE
Create an empty lock file when the file is opened so that cloud drives can detect it and lock the file to the user editing it locally.

### ISSUE THREAD

**Issue by vagner-dias** [ONLYOFFICE/DesktopEditors#1413]
### This issue is unique.

- [X] I have used the [search tool](https://github.com/ONLYOFFICE/DesktopEditors/issues?q=) and did not find an issue describing my idea.

### Your idea.

Cloud services such as NextCloud, Google Drive, OneDrive, SeaFile, etc., have built-in mechanisms to detect "Office Lock files" (~$ Files) so that they can lock these files to the user who is editing them locally, to avoid conflicts.

The **NextCloud version of OnlyOffice already works perfectly with file locking** for web editing; however, the **desktop version lacks this feature**.

In short, the implementation would consist of creating **an empty lock file** (~$FileName.Extension) every time a file is opened in the desktop editor, and when the file is closed, this lock file would be deleted.

That way, these cloud drives would **work better with OnlyOffice**.

Regards.

**Comment by vagner-dias** [comment]
> Hi @vagner-dias! Sorry for the long answer, are you talking about desktop cloud storage clients and files that are created locally?

Hi @LuckySpecialist , exactly.

**Comment by Rita-Bubnova** [comment]
We are reopening the task.
We will close the task as soon as the problem is fixed.

**Comment by maxsnts** [comment]
While this is probable not one of your concerns, it would be amazing if you could use/check the lock files created by LibreOffice.
This way, maybe when other people is using LibreOffice, any of the Apps would create the same lock file and avoid destroying the files.

Thank you
Best regards

**Comment by LuckySpecialist** [comment]
The problem has been fixed since version 9.1, and this task can be closed.

**Comment by Rita-Bubnova** [comment]
I close this issue. Feel free to comment or reopen it if you got further questions.
